src/components/MemberSearchPage/MemberIntersts.js

```diff
@@ -30,6 +30,10 @@ class MemberInterests extends Component {
       filter: !prevState.filter,
     }));
   }
+  @bind
```

Comment:
should be removed?

Reply:
yes i seen it a couple of minutes after submitting the PR...